packages/react-signer/package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
     "@polkadot/hw-ledger": "^12.6.2",
     "@polkadot/react-components": "^0.133.2-81-x"
+    "azero-wallet-adapter": "^0.3.1"
```

Comment:
There is comma missing, could you fix it? The file should be generate but yarn anyway, right?

Reply:
Hmm I don't see this comma missing in my local copy, so I think the file got regenerated.